os/dbuild.sh

```diff
@@ -109,6 +109,9 @@ function SELECT_OPTION()
 			echo "  \"4. Clean Build\""
 			echo "  \"5. Clean Build and Re-Configure\""
 			echo "  \"6. Build SmartFS Image\""
+			if [ "${STATUS}" == "BUILT" ]; then
```

Comment:
I dont think it is necessary to check for built status. User can run the trap tool from a previous build or from a binary shared by tester.